### ISSUE TITLE
Autoconf version compatibility

### ISSUE THREAD

**Issue by j-woz** [swift-lang/swift-t#195]
Under Autoconf 2.71 (Ubuntu 22) 'autoconf -f' does not produce all needed auxiliary files, resulting in:
```
swift-t/c-utils/code> ./bootstrap 
Running autoconf...
Running autoheader...
ExM C-UTILS: bootstrap: OK
 swift-t/c-utils/code> ./configure --config-cache --prefix=...
configure: loading cache config.cache
configure: error: cannot find required auxiliary files: config.guess config.sub
```

**Comment by j-woz** [comment]
Fixed by 0f8459161

**Comment by j-woz** [comment]
This was tested with Ubuntu 18 and 22, autoconf 2.69 and 2.71 .
